### ISSUE TITLE
Automatic Release management

### ISSUE THREAD

**Issue by bneumann** [bneumann/CppUTest-Test-Adapter#39]
I do the releases manually currently but I always struggle with the release of new packages. I started a github action to automate it, but don't have that much time at the moment. There is a branch for that for testing around, feel free to fiddle with it and get it done :)

**Comment by alastairtree** [comment]
@bneumann thanks for maintaining this useful tool. I am guessing this issue is the reason the latests release is 1.3 and the one on vscode gallery is 1.1 still? https://marketplace.visualstudio.com/items?itemName=bneumann.cpputest-test-adapter

**Comment by bneumann** [comment]
Haha yes ;) I tried to make the github action listen to taggings. But a push of npm version did not trigger the build.
I think the 1.1.1 build was when I triggered the action manually. So technically the automated publishing works with PAT secret and all, but not the triggering. 
I know a bit about gitlab and Azure pipelines but haven't yet had the time to really dig into github actions. So if you have some experience feel free to give it a spin
